shift/wizard/assign_super_coop.py

```diff
@@ -2,8 +2,8 @@
 
 
 class AssignSuperCoop(models.TransientModel):
-    _name = "beesddoo.shift.assign_super_coop"
```

Comment:
🙄 at the previous name ...

In the new name, did you consider replacing underscores by dots ?

Reply:
Not replacing the underscores with dots this late in the PR. It's probably fine.